virtualizarr/tests/test_xarray.py

```diff
@@ -878,3 +880,223 @@ def test_drop_variables(netcdf4_file):
         file_url=netcdf4_file, object_store=store, parser=parser, drop_variables=["air"]
     ) as vds:
         assert "air" not in vds.variables
+
+
+def test_xarray_concat_empty_data_var(tmp_path: Path):
```

Comment:
This test seems extremely long and complicated to test such a tiny change. Couldn't we just test this with something like

```python
marr = create_manifest_array(shape=())
marr[indexer]
```

Reply:
I simplified the tests, and checked that the new tests would fail without the bugfix here.